CHANGES

```diff
@@ -22,12 +22,16 @@ description indicates whether it is SOURCE-COMPATIBLE, BINARY-COMPATIBLE, or BRE
 Further information about [seL4 releases](https://docs.sel4.systems/sel4_release/) is available.
 
 ---
-Upcoming release: BINARY COMPATIBLE
+Upcoming release: BREAKING
 
 ## Changes
 
  * Added support for the ARM Cortex A55
  * Added support for the ODroid C4
+ * Changed kernel boot parameters to become architecture agnostic with using word_t instead of uint32. Stop using
+   sword_t due to C rule violations, use word_t instead and explicitly allow overflows and underflows in the
```

Comment:
`sword_t` never made it into a permanent branch, so the reference to it in the comment is probably confusing.

Reply:
sword_t is present in this repository.  It just never made it into the elfloader repository.

Reply:
Oh yes, you're right. I double checked and still managed to miss it, sorry. All good then.